src/components/substrate/SubstrateTxButton.tsx

```diff
@@ -186,31 +211,61 @@ function TxButton({
     doOnFailed(null)
   }
 
-  const sendSignedTx = async () => {
-    if (!accountId) {
-      throw new Error('No account id provided')
-    }
+  const sendOffchainTx = async (extrinsic: SubmittableExtrinsic, offchainToken: string) => {
```

Comment:
Looks like you can extract the function from component

Reply:
extracted in 7708dbd